app/__init__.py

```diff
@@ -23,9 +28,11 @@
 app.config.update(CELERYBEAT_SCHEDULE={
     'say-every-5-seconds': {
         'task': 'store',
-        "schedule": 86400.00,
-         }
+        "schedule": crontab(hour='*/24'),
+    }
 })
+session_token = ''
```

Comment:
what does this mean? why are you declaring it with empty strings?

Reply:
initialized that string because it is used in views.py file in conditional statements.